source/Logger.cpp

```diff
@@ -370,6 +370,7 @@ void FPSciLogger::logTargetTypes(const Array<shared_ptr<TargetConfig>>& targets)
 			"'" + config->destSpace + "'",
 			String(std::to_string(config->size[0])),
 			String(std::to_string(config->size[1])),
+			String(std::to_string(config->symmetricEcc)),
```

Comment:
You need to add this column to the table creation at line 75 above in this file for this to work